plugins/thumbnail/lib/model/enum/kThumbnailParameterName.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @package plugins.thumbnail
+ * @subpackage model.enum
+ */
+interface kThumbnailParameterName extends BaseEnum
+{
+	const WIDTH = "width";
+	const HEIGHT = "height";
+	const BACKGROUND_COLOR = "background_color";
```

Comment:
Keep same nameing method for all,
Either namea_nameb or nameaNameb such as in COMPOSITE_FIT